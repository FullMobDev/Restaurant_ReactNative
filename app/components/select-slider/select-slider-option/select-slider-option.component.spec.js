import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow } from 'enzyme';
import React from 'react';

import SelectSliderOption from './select-slider-option.component';

Enzyme.configure({ adapter: new Adapter() });

describe('<SelectSliderOption>', () => {
    // test('displays option as label', () => {
    //     const wrapper = shallow(
    //         <SelectSliderOption
    //             option={'option1'} 
    //             selectPositionX={0}
    //         />
    //     );
    //     expect(wrapper).toMatchSnapshot();
    // });
});
