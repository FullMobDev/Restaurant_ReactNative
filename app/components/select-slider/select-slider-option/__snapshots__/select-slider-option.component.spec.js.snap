// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectSliderOption> displays option as label 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    option1
  </Text>
</View>
`;
